src/index.html

```diff
@@ -2,21 +2,30 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Stop watch</title>
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
You should include CSS files into HTML, not SCSS. SCSS files are preprocessed to CSS and the resulting CSS is what should be linked in your HTML. Make sure to compile your SCSS to CSS and link to the CSS file instead.